keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -0,0 +1,270 @@
+
+### Goals
+
+See [User Stories](#user-stories).
+
+<unresolved>
+
+### Non-Goals
+
+- Prescribing an issue triage workflow for all projects within Kubernetes orgs
+
+</unresolved>
+
+## Proposal
+
+### User Stories
+
+#### Group Leads
+
+As a SIG Chair/Technical Lead, WG/UG organizer, or subproject owner, I want to
+be able to easily triage my group's backlog of open issues and PRs to accelerate
+merge velocity and issue resolution.
+
+#### Reviewers and Approvers
+
+As a reviewer/approver, I want a simple system to categorize issues/PRs in my
+purview, so that I can prioritize what to review first.
+
+#### Contributors
+
+As a contributor, I want to be able to submit issues or PRs and:
+
+- understand how the community works to address new submissions
+- have some assurance that they will be routed to the correct group
+- have them addressed in a timely manner
+
+<unresolved>
+
+### Notes/Constraints/Caveats (optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above.
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they releate.
+-->
+
+</unresolved>
+
+<unresolved>
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal and how do we mitigate.  Think broadly.
+For example, consider both security and how this will impact the larger
+kubernetes ecosystem.
+
+How will security be reviewed and by whom?
+
+How will UX be reviewed and by whom?
+
+Consider including folks that also work outside the SIG or subproject.
+-->
+
+</unresolved>
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable.  This may include API specs (though not always
+required) or even code snippets.  If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+### Graduation Criteria
+
+<!--
+**Note:** *Not required until targeted at a release.*
+-->
+
+_Not required in `provisional` state._
+
+### Phase 0
+
+#### needs-triage and triage/accepted-by-sig labels
+
+We currently have a Prow plugin called `require-matching-label`, which requires
+specific labels to be set on a issue or pull request.
+
+Examples in [kubernetes/kubernetes] include:
+
+- `needs-kind` (needs a `kind/foo` label)
+- `needs-priority` (needs a `priority/bar` label)
+- `needs-sig` (needs a `sig/baz` label)
+
+Here we propose [introducing two new labels](https://github.com/kubernetes/test-infra/pull/16298):
+
+- `needs-triage`
+- `triage/accepted-by-sig`
+
+The `needs-triage` label would be automatically applied to incoming issues.
+
+Current contributors would then need to inspect issues with the `needs-triage`
+label and attempt to triage them.
+
+Upon determining that an issue is ready to be actively worked on, an org member
+can apply the `triage/accepted-by-sig` label using the following bot command:
+
+```shell
+/triage accept
```

Comment:
I see that is shorter than what I wrote above.  I'm going to leave the bikeshed - my gift to you.

Reply:
I hope you have a gift receipt! :upside_down_face: